dashboard/modules/job/job_description.py

```diff
@@ -0,0 +1,58 @@
+import attr
+from attr.validators import instance_of, in_, deep_mapping
+from ray.core.generated import common_pb2
+
+
+@attr.s(repr=False, slots=True, hash=True)
+class _AnyValidator(object):
+    def __call__(self, inst, name, value):
+        pass
+
+    def __repr__(self):
+        return "<any validator for any type>"
+
+
+def any_():
+    return _AnyValidator()
+
+
+@attr.s(kw_only=True, slots=True)
+class RuntimeEnv:
+    # The url to download the job package archive. The archive format is
+    # one of “zip”, “tar”, “gztar”, “bztar”, or “xztar”. Please refer to
+    # https://docs.python.org/3/library/shutil.html#shutil.unpack_archive
+    workingDir = attr.ib(type=str, validator=instance_of(str))
```

Comment:
We should use the existing definition for this please:
https://github.com/ray-project/ray/blob/master/python/ray/_private/runtime_env.py#L29

Reply:
Is the existing definition validates the data?

Reply:
There is some validation but we could use some more. If we need more, let's add it there so we have a single definition.

Reply:
Can I replace the https://github.com/ray-project/ray/blob/master/python/ray/_private/runtime_env.py#L29 with this type? This `RuntimeEnv` type validates the RESTful post data which is a JSON in google style (https://google.github.io/styleguide/jsoncstyleguide.xml), so the attributes are Camel-Case.

Reply:
We should definitely be using snake_case for the fields to keep it pythonic.